pkg/shared/remote/DynamicServiceConstants.go

```diff
@@ -19,4 +19,5 @@ type AmsConfig struct {
 	TrialProductQuotaID         string `json:"trialQuotaProductId"`
 	LongLivedQuotaProductID     string `json:"longLivedQuotaProductId"`
 	ResourceName                string `json:"resourceName"`
+	EnterpriseProductQuotaID    string `json:"RHOSAKCC"`
```

Comment:
`EnterpriseProductQuotaID    string `json:"enterpriseProductQuotaID"``

Reply:
This has to be added in here as well: https://github.com/dimakis/app-services-cli/blob/b0164e5ea4ac899e8b35d3b8a0972833f86e92b6/pkg/shared/remote/service-constants.json#L4

Reply:
`"enterpriseProductQuotaID": "RHOSAKCC"`

Reply:
damn I was missing it in the json file, I was wondering why it wasn't working. thank you 😄